Homework3/Menu.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+
+namespace start
+{
+    public static class Menu
+    {
+        public static void Run()
+        {
+            Console.WriteLine("Please choose:");
+            Console.WriteLine("1. Login.");
+            Console.WriteLine("2. Register new user.");
+            Console.WriteLine("3. Exit program.");
+
+            switch (ReturnValidUserInput())
+            {
+                case 1:
```

Comment:
could replace magic numbers with constants. In that case- changing the options would be just a matter of changing the constant.

Reply:
IN one place, as opposed to many

Reply:
I made constants instead and used their names. Hopefully I understood this correctly?

Reply:
Yup, all good 👍